src/pal/src/CMakeLists.txt

```diff
@@ -68,7 +68,13 @@ endif()
 # turn off capability to remove unused functions (which was enabled in debug build with sanitizers)
 set(CMAKE_SHARED_LINKER_FLAGS_DEBUG "${CMAKE_SHARED_LINKER_FLAGS_DEBUG} -Wl,--no-gc-sections")
 
-add_compile_options(-fno-builtin)
+# NetBSD requires a compiler (built-in) version of alloca(3)
+if(NOT CMAKE_SYSTEM_NAME STREQUAL NetBSD)
+  add_compile_options(-fno-builtin)
```

Comment:
@krytarowski Could you please try to apply the NetBSD version to all Unixes? (It would be nice if we can make this whole thing simpler.)